client/src/app/site/pages/meetings/pages/participants/export/participant-csv-export.service/participant-csv-export.service.ts

```diff
@@ -57,8 +60,37 @@ export class ParticipantCsvExportService {
         );
     }
 
-    public exportCsvExample(): void {
+    /**
+     * @param meeting
+     * @returns participants csv-example with added 'groups' value:
+     * - 2 custom group (not default or admin) names separated by comma by default
+     * - 1 custom group name if meeting has only 1 custom group
+     * - default group name if meeting has no custom groups
+     */
+    private addParticipantGroups(): UserExport[] {
```

Comment:
Mmh, the name 'addParticipantGroups" is misleading, it should be like eg 'provideRows'...